src/Components/IndexScene/ShowLogsButtonScene.tsx

```diff
@@ -0,0 +1,80 @@
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { Button, useStyles2 } from '@grafana/ui';
+import React from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { navigateToInitialPageAfterServiceSelection } from '../../services/navigate';
+import { getLabelsVariable } from '../../services/variableGetters';
+import { FilterOp } from '../../services/filterTypes';
+import { testIds } from '../../services/testIds';
+
+export interface ShowLogsButtonSceneState extends SceneObjectState {
+  disabled?: boolean;
+  hide?: boolean;
```

Comment:
Just for consistency, `disabled` is past tense, `hide` isn't. Should we change to `hidden`?